epub33/rs/index.html

```diff
@@ -862,7 +862,7 @@ <h3>Scripting</h3>
 							provided to scripts during execution (e.g., limiting networking).</p>
 					</li>
 					<li>
-						<p id="confreq-rs-scripted-readingsystem-object">It MUST extend the <a
+						<p id="confreq-rs-scripted-readingsystem-object" data-tests="#scripting-epubReadingSystem-features,#scripting-epubReadingSystem-support,#scripting-epubReadingSystem-support_svg">It MUST extend the <a
```

Comment:
I'll comment on the names more in the other PR, but a) I think we should stick with 3-letter abbreviations for consistency (I guess scr-) and b) as below, I don't think we should introduce mixed case in names.

I recommend #scr-readingsystem-features, etc.

Applies throughout.